packages/sso-express/package.json

```diff
@@ -26,21 +26,18 @@
     "express": "4.x",
     "openid-client": "5.x"
   },
-  "dependencies": {
-    "ts-jest": "^27.0.7"
-  },
   "devDependencies": {
     "@types/express": "^4.17.13",
     "@types/jest": "^27.0.3",
     "@types/node": "^16.11.9",
     "express": "4.x",
     "jest": "^27.3.1",
     "openid-client": "^5.0.2",
+    "ts-jest": "^27.0.7",
     "typescript": "^4.5.2"
   },
   "files": [
-    "ssoExpress.js",
-    "helpers.js"
+    "dist/"
   ],
   "gitHead": "2eae856a51caef7c61664d15ee25bea0dfc844db"
```

Comment:
This githead can go, it's a relic from Lerna when I did manual publishes a long time ago

Reply:
the link to that commit is ancient